src/main/java/com/opentable/extension/BodyTransformer.java

```diff
@@ -177,6 +181,20 @@ private CharSequence getValueFromRequestObject(String group, Map requestObject)
         return String.valueOf(tempObject);
     }
 
+    private CharSequence replaceValueFromRequestObject(String group, Map requestObject) {
```

Comment:
This is code duplicate of `getValueFromRequestObject` method. This method should start from using the previous one to get field value and them apply required modifications. Please make it more generic or add one more layer of abstraction